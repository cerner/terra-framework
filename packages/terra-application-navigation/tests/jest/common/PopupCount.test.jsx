import React from 'react';
import PopupCount from '../../../src/common/_PopupCount';

describe('PopupCount', () => {
  it('should render default element', () => {
    const shallowComponent = shallow(
      <PopupCount />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });

  it('should render prop data', () => {
    const shallowComponent = shallow(
      <PopupCount
        value={1}
      />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });
});
