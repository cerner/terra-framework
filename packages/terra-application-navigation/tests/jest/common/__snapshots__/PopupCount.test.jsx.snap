// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopupCount should render default element 1`] = `
<div
  className="popup-count"
/>
`;

exports[`PopupCount should render prop data 1`] = `
<div
  className="popup-count"
>
  1
</div>
`;
